### DISCUSSION TITLE
在github里搜“clash android”，搜不到本项目

### DISCUSSION THREAD

**Discussion by luo-chuan** [MetaCubeX/ClashMetaForAndroid#314] [General]
我还以为这个项目没了。

**Comment by whengu** [comment]
使用github搜索功能, 以关键字 "clash meta for android" 和 "clash meta" 也搜索不到此项目, 但是用仓库链接可以进入仓库主页, 不知道是不是IP的问题

**Comment by fejeIaqbhbv** [comment]
因为github默认不会把fork的仓库放进搜索结果

有需要的在搜索时加入 `fork:true`

https://docs.github.com/search-github/searching-on-github/searching-in-forks

![s](https://github.com/user-attachments/assets/668386b7-0697-447c-902e-305f39be68f8)

**Reply by BIGlettuces** [reply]
一般收藏再说....就这么个唯二的独苗了......😂

**Reply by veratione** [reply]
回复一下，让更多人看见。

**Reply by a447948814** [reply]
马克

**Comment by 1851428779** [comment]
仓库改名了，搜内核关键词mihomo就能搜到
